.github/workflows/tests.yml

```diff
@@ -155,8 +155,9 @@ jobs:
 
       - name: Pin provider versions
         run: |
-          sed -i 's/>=\(.*# tftest\)/=\1/g' default-versions.tf
-          find -name versions.tf -exec cp default-versions.tf {} \;
+          for f in $(find . -name versions.tf); do
+            sed -i 's/>=\(.*# tftest\)/=\1/g' $f;
+          done
```

Comment:
Horrible horrible.
For the record: `gfind . -name versions.tf -print0 | xargs -0 gsed -i 's/>=\(.*# tftest\)/=\1/g'`

Reply:
So basically Perl FTW 😂